src/main/java/Board.java

```diff
@@ -1,4 +1,105 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class Board {
-    public static String[] grid = new String[9];
+
+    private List<String> cells = Arrays.asList("", "", "", "", "", "", "", "", "");
+
+    public List<String> getCells() {
+        return cells;
+    }
+
+    public boolean gridIsEmpty() {
+       List<String> currentBoard = getCells();
+       for (int i = 0; i < currentBoard.size(); i++) {
+           if (currentBoard.get(i) != "") {
+               return false;
+           } else {
+               continue;
+           }
+       } return true;
+    }
+
+    public List<String> markCell(int index, String marker) {
+        List<String> currentGrid = getCells();
+        currentGrid.set(index, marker);
+        return currentGrid;
+    }
+
+    public boolean availableCell(int index) {
+        List<String> currentGrid = getCells();
+        String cell = (String) currentGrid.get(index);
```

Comment:
I don't think you need to explicitly cast to a String here. Because you are using generics `List<String>`, the code knows the list contains strings.